templates/cemanager.yaml

```diff
@@ -120,11 +120,6 @@ spec:
                 secretKeyRef:
                   name: built-in-envproxy-token
                   key: token
-              # ENVIRONMENT_SERVICE_ACCOUNT is the service account to assign
```

Comment:
this comment is in another place already (environment.yaml) so it's kinda redundant anyways